### PR TITLE
mark password as a sensitive param for 8.2

### PR DESCRIPTION
**Pull request by remicollet**
Notice: this imply to use **8.2** to generate arginfo.

### CODE REVIEW

File: php_memcached.c [hunk 1]
```diff
@@ -3895,9 +3895,14 @@ PHP_METHOD(MemcachedServer, on)
 
 #endif
 
+#if PHP_VERSION_ID < 80200
```
Comment (remicollet) [inline php_memcached.c:3898]: this should be defined only for 8.0/8.1, but needed for 7.x because of https://github.com/php/php-src/issues/8842

**Review by sodabrew — APPROVED** [review]
lgtm!

**Comment by TimWolla** [comment]
Note: You'll likely need to update this PR due to php/php-src@342e18f10518d3dd5381fcdd8e84da84d1a8540c which removes the `zend_mark_function_parameter_as_sensitive()` function in favor of a more generic solution.

**Comment by remicollet** [comment]
Refreshed for upcoming 8.2.0beta1

**Comment by remicollet** [comment]
CI is not related to this PR, so suashed and merged

**Pull request merged**